.cirrus.yml

```diff
@@ -166,7 +166,9 @@ task:
     - name: downgraded_analyze
       depends_on: analyze
       analyze_script:
-        - ./script/tool_runner.sh analyze --downgrade --custom-analysis=script/configs/custom_analysis.yaml
+        # TODO(chunhtai): Includes go_router_builder once downgraded_analyze picks up the latest
+        # change https://github.com/flutter/flutter/issues/114106.
```

Comment:
I don't understand what this comment means; downgraded analyze uses the lowest version your pubspec allows. The only way to change what version it gets is to change your pubspec, so I don't see what the TODO is here that can't just be done right now instead (or if it can't, why not, and what the plan is for changing that).

Reply:
ah you are right let me try to bump up the analyzer version

Reply:
you are right, bump the pubspec yaml version